package.json

```diff
@@ -43,6 +43,7 @@
     "@vaadin/testing-helpers": "^0.2.1",
     "@web/dev-server": "^0.1.17",
     "@web/test-runner": "^0.13.4",
+    "@web/test-runner-commands": "^0.4.5",
```

Comment:
Added this plugin to be able to send actual keyboard inputs in tests. Our current approach of creating keyboard events does not trigger the default browser action, such as switching focus.

A much bigger task here would be to upgrade existing tests to use this instead of the event-based approach, however that's a huge change as all tests using keyboard inputs need to be adapted to use the async API of the plugin.